dist_example.py

```diff
@@ -151,66 +91,93 @@ def test(args, model, device, test_loader, verbose=True):
 
     test_loss /= len(test_loader.dataset)
 
-    acc = correct / len(test_loader.dataset)
+    acc = correct / len(test_loader.dataset) 
     if verbose:
         msg = "\nTest set: loss_avg={:.3f}, accuracy=({:.1f}%)\n"
         print(msg.format(test_loss, 100.0 * acc))
     return {"acc": acc, "loss": test_loss}
 
+def train_model(model, train_set, kwargs):
+    """
+    Trains model using an adaptive batch size
```

Comment:
I'd document this function better. What type are `model`, `train_set` and `kwargs`? What keys are expected in `kwargs`?